options/options.go

```diff
@@ -0,0 +1,5 @@
+package options
+
+type DeleteOption interface {
```

Comment:
Go-doc with examples on how to implement custom delete options might be handy here.

Reply:
Added go doc https://github.com/C2FO/vfs/pull/119/commits/6bde3b81feda5219229324dd9c0ccc3a06a66a30#diff-76f121a3a81c8945dc0e0c81a4b95e941d252522eba8704a1aa6f6a1722a8b68R3-R13